test/test_dtype.py

```diff
@@ -433,6 +453,7 @@ def test_dtype_str_arg(self):
   @given(strat.sampled_from(dtype_ints), strat.sampled_from(dtype_floats))
   def test_creation(self, default_int, default_float):
     dtypes.default_int, dtypes.default_float = default_int, default_float
+    if default_float in (dtypes.fp8_e4m3, dtypes.fp8_e5m2): raise unittest.SkipTest("no test_creation for fp8")
```

Comment:
Why?

Reply:
Need to adapt those. Some creation methods are not exact vs float since some numbers are not representable in fp8_e4m3 or fp8_e5m2.

Reply:
resolved